karapace/master_coordinator.py

```diff
@@ -49,16 +50,25 @@ def group_protocols(self):
 
     def _perform_assignment(self, leader_id, protocol, members):
         self.log.info("Creating assignment: %r, protocol: %r, members: %r", leader_id, protocol, members)
-        self.master = None
+        self.are_we_master = None
         error = NO_ERROR
         urls = {}
+        fallback_urls = {}
```

Comment:
I know you probably want to get this merged, just one more thing. But we don't need this, `fallback_urls` doesn't have any `master_eligibility` nodes in it, so these URLs are not really used for fallback.

Reply:
AFAIU `_perform_assignment` should return Kafka assignment. `fallback_urls` is used for returning assignment even if not setting it as master. Thus `fallback_urls` was named just based on `urls`. Do you have suggestions for better naming?

Reply:
My bad, I got tunnel vision here and forgot about the `assignments` below.